e2e/libs/volume/crd.py

```diff
@@ -214,7 +214,6 @@ def wait_for_volume_state(self, volume_name, desired_state):
             except Exception as e:
                 logging(f"Getting volume {self.get(volume_name)} status error: {e}")
             time.sleep(self.retry_interval)
-        assert self.get(volume_name)["status"]["state"] == desired_state
```

Comment:
Is there a specific reason for the removal of the final assertion? Same below.

Reply:
Refined. Thanks!